### ISSUE TITLE
Digest Page - Research for Newsletter Signup

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-energynewsnetwork#63]
A consolidated page for MailChimp subscriptions to allow users to manage subscriptions.

**Comment by benlk** [comment]
Does MailChimp not already provide this to subscribers, via a link in Mailchimp-sent emails?

Or is the idea that this would live on energynews.us?

**Comment by benlk** [comment]
Maybe this link? https://mailchimp.com/help/find-unsubscribe-link-audience/

But that only works if they're all in one Audience.

**Comment by benlk** [comment]
Questions that require Mailchimp access to research:

- [ ] are they all in one audience?
- [ ] if so, is there a single manage subscription link that's available to people in that audience?
- [ ] if not, are there alternatives?

**Comment by benlk** [comment]
- ENN is migrating everyone to a single audience
- currently asking MC if there's a single public link to the manage-subscription page

**Comment by benlk** [comment]
There is not a single public link to the manage-subscription page.

> Institute for Nonprofit News:
> There's a public unsubscribe link for an audience: https://mailchimp.com/help/find-unsubscribe-link-audience/
> 
> Is there a public link for the "Update Profile" page? All I can see in Mailchimp's docs is a way to add that link as a merge tag in an email: https://mailchimp.com/help/add-an-update-profile-link/

> Dorothy
> Hi there, how are you today? I'm reading your question
> Gotcha! There is not a public update profile link, though I can see how that would be useful, and I'll go ahead and pass along the feedback for you. 
> 
> The only way someone could access the update profile link outside an email campaign is if an existing subscriber filled out a Mailchimp form, such as the hosted form or embed code form. From there, we'd display a message that they are already subscribed, and that they can click the provided link to update their profile. We'd then send them an email (for security reasons) where they can update their profile

> Institute for Nonprofit News
> So the update-profile link described in https://mailchimp.com/help/add-an-update-profile-link/ isn't a fixed link, but instead something dynamic that varies by subscriber?

> Dorothy
> Yes, that's correct. The update profile link takes people to a unique URL that displays their profile information. Because of this, we don't have a generic link that people can use, similar to the unsubscribe form.

> Institute for Nonprofit News
> Okay, this is good to know.
> I'm going to restate what you said to make sure I understand it:
> 
> If we want to have a place on our website where people could manage their own subscription, what that will look like is an instruction to instead fill out the subscribe form again, using the same email address they used to sign up. When they fill it out, they'll get a link from Mailchimp saying they've already signed up, and will receive an email with a link to their user-specific manage-profile page.
> Is that all right?

> Dorothy
> Yep! That's exactly right. You'll just want to make sure the signup form is one that we profile, such as the hosted form, or the embed code form. You can also direct them to the footer of your emails, as it should include a link to update their profile as well

**tl;dr:** we can't provide a button to take people to "Manage your subscription".

<img width="265" alt="Screen Shot 2020-02-10 at 18 02 18" src="https://user-images.githubusercontent.com/1754187/74198479-7ea2e500-4c2f-11ea-9090-0ba122897b52.png">

**Comment by kaylima** [comment]
@benlk  can we design the field to be something like:
"Enter the email used to subscribe" and then "thank you, check your email for instructions on how to manage your subscriptions"... or something more condensed? cc @MirandaEcho

**Comment by MirandaEcho** [comment]
The page is actually called "manage profile", and can be added with a merge tag to any email list, so wouldn't we be able to copy that link and use that?

**Comment by MirandaEcho** [comment]
Deleted my last comment after seeing the chat from MailChimp.  The approach Kay outlined makes sense, would that work?

**Comment by benlk** [comment]
So here's the code of a standard signup form for this audience:

```html
<!-- Begin Mailchimp Signup Form -->
<link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
<style type="text/css">
	#mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
	/* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
</style>
<div id="mc_embed_signup">
<form action="https://energynews.us7.list-manage.com/subscribe/post?u=ae5d3a0c6088cad29d71bf0d0&amp;id=efa0033ba9" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
    <div id="mc_embed_signup_scroll">
	<h2>Subscribe</h2>
<div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
<div class="mc-field-group">
	<label for="mce-EMAIL">Email Address  <span class="asterisk">*</span>
</label>
	<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
</div>
<div class="mc-field-group input-group">
    <strong>Subscribe to Updates </strong>
    <ul><li><input type="checkbox" value="1" name="group[6829][1]" id="mce-group[6829]-6829-0"><label for="mce-group[6829]-6829-0">Midwest Energy News</label></li>
<li><input type="checkbox" value="2" name="group[6829][2]" id="mce-group[6829]-6829-1"><label for="mce-group[6829]-6829-1">Southeast Energy News</label></li>
<li><input type="checkbox" value="4" name="group[6829][4]" id="mce-group[6829]-6829-2"><label for="mce-group[6829]-6829-2">National Updates</label></li>
<li><input type="checkbox" value="8" name="group[6829][8]" id="mce-group[6829]-6829-3"><label for="mce-group[6829]-6829-3">Northeast Energy News</label></li>
<li><input type="checkbox" value="16" name="group[6829][16]" id="mce-group[6829]-6829-4"><label for="mce-group[6829]-6829-4">Southwest Energy News</label></li>
</ul>
</div>
	<div id="mce-responses" class="clear">
		<div class="response" id="mce-error-response" style="display:none"></div>
		<div class="response" id="mce-success-response" style="display:none"></div>
	</div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_ae5d3a0c6088cad29d71bf0d0_efa0033ba9" tabindex="-1" value=""></div>
    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
    </div>
</form>
</div>
<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='MMERGE3';ftypes[3]='text';fnames[4]='MMERGE4';ftypes[4]='text';fnames[16]='MMERGE16';ftypes[16]='text';fnames[17]='MMERGE17';ftypes[17]='text';fnames[18]='MMERGE18';ftypes[18]='text';fnames[19]='MMERGE19';ftypes[19]='text';fnames[20]='MMERGE20';ftypes[20]='text';fnames[21]='MMERGE21';ftypes[21]='text';fnames[22]='MMERGE22';ftypes[22]='text';fnames[23]='MMERGE23';ftypes[23]='text';fnames[24]='MMERGE24';ftypes[24]='text';fnames[25]='MMERGE25';ftypes[25]='text';fnames[5]='INACTIVE';ftypes[5]='number';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
<!--End mc_embed_signup-->
```

The key bit is `<div class="response" id="mce-success-response" style="display:none"></div>`

It makes sense to me that we could put the message there, but we won't know until we try it.

**Comment by joshdarby** [comment]
So this actually requires a few more steps from the user than just
- fill out form
- get email to manage subs

What it actually looks like is
- fill out form
- form opens link to page that looks like this
![Screen Shot 2020-04-10 at 9 58 11 AM](https://user-images.githubusercontent.com/18353636/78996090-de274f00-7b11-11ea-826d-1e4a8fe11e80.png)
- user must click "click here to update your profile"
- user then gets an email with a link to update their profile
- user clicks link in email and then can update their subs


@MirandaEcho @kaylima Is that still an ok process?

Also, not sure why the page you get taken to after filling out the form has those fields to fill out, but they don't actually appear to work 🤔

**Comment by joshdarby** [comment]
The sidebar on staging has been updated with a manage subscription form. It can be tested here https://usenergystage.wpengine.com/category/digest/

![Screen Shot 2020-04-10 at 10 16 23 AM](https://user-images.githubusercontent.com/18353636/78997340-61e23b00-7b14-11ea-9b2e-a842c0bf33b2.png)
